docs/source/usage/generating_metaframes.rst

```diff
@@ -41,11 +43,15 @@ Let's take a look at each optional parameter individually:
 
 spec
 ^^^^
-**spec** is an optional dictionary that outlines specific directives for each DataFrame column. The potential directives include:
+**spec** is an optional dictionary that outlines specific directives for each DataFrame column (variable). The potential directives include:
    
     - ``distribution``: Allows you to specify the statistical distribution of each column. To see what distributions are available refer to the :doc:`distribution package API reference</api/metasynth.distribution>`.
     
-    - ``unique``: Declare whether the column in the synthetic dataset should contain unique values.
+    - ``unique``: Declare whether the column in the synthetic dataset should contain unique values. 
+    
+    .. admonition:: Detection of unique variables
+
+        MetaSynth automatically analyzes the DataFrame for columns that contain unique values, and notify the user with a warning when it detects one. This warning can safely be ignored. Alternatively, you can set the column to unique in the ``spec`` parameter to ensure that MetaSynth only produces unique values for this variable and remove the warning. 
```

Comment:
Instead of "safely ignored", perhaps we can say here what actually happens when this warning happens (unique is set to False).